lms/product/d2l/_plugin/misc.py

```diff
@@ -6,6 +6,10 @@ class D2LMiscPlugin(MiscPlugin):
     def __init__(self, create_line_item: bool):
         self._create_line_item = create_line_item
 
+    # Deep linking in D2L implies creating a new assignment.
+    # Prompt for a title to set it for the new assignment.
+    deep_linking_prompt_for_title = True
```

Comment:
This will belong to a new DL plugin or similar abstraction once all of this is done.